src/kata1.fizzBuzz.js

```diff
@@ -1,5 +1,13 @@
 const fizzBuzz = (number) => {
-
+ if (number === 3){
```

Comment:
3, 6, 9, 12 etc should all return fizz because they are all multiples of 3.
5, 10, 20, 25 should all return buzz because they are all multiples of 5.
15, 30, 45 should all return fizzbuzz because they are all multiples of 3 and of 5.
Your code will only return fizz for 3, buzz for 5 and fizzbuzz for 8